contributions/catalog/webapps/templates/contribute/capability_details.html

```diff
@@ -30,156 +26,175 @@
     <h1>Capability Details</h1>
 
     <hr class="divider"/>
-    <div class="row">
-        <div class="col-12 col-sm-2">
-            <h4>Capability Name </h4>
-        </div>
-        <p class="col-sm-9">{{post.name}}</p>
-    </div>
-
-    <div class="row">
-        <div class="col-12 col-sm-2">
-            <h4>Capability description </h4>
-        </div>
-        <p class="col-sm-9">{{post.description}}</p>
-    </div>
-
-    <div class="row">
-        <div class="col-12 col-sm-2">
-            <h4>Is code open source?</h4>
-        </div>
-        <p class="col-sm-9">{{post.isOpenSource}}</p>
-    </div>
-
-    <div class="row">
-        <div class="col-12 col-sm-2">
-            <h4>Source code repository URL</h4>
+    <div class="grid container" style="display: grid; grid-auto-rows: max-content; row-gap: 5px; grid-template-rows: fit-content(100%)">
```

Comment:
Hi, @minump, I got a better alignment with the following. Could you please take a look when you get a chance?

```
<div class="container">
        <div class="row">
            <div class="col-12 col-md-3">
                <h4>Capability Name </h4>
            </div>
            <div class="col-12 col-md-9">
                <div class="small-body-text">{{post.name}}</div>
            </div>
        </div>


        <div class="row">
            <div class="col-12 col-md-3">
                <h4>Capability description </h4>
            </div>
            <div class="col-12 col-md-9 ">
                <div class="small-body-text">{{post.description}}</div>
            </div>
        </div>
```

Similarly:

```
.small-body-text {
    font-size: 14px;
    height: 16px;
    font-weight: normal;
    font-stretch: normal;
    font-style: normal;
    letter-spacing: normal;
    margin: 15px auto;
    text-align: left;
    color: #404040;
}
```

Reply:
Updated.

Reply:
With small body text ```margin : 11px auto;``` , the key and value does not seem to be in line (see screenshot). When its just ```margin :auto;``` it seems to be aligned.
Updated the rest of the styling as per feedback.
![Screen Shot 2022-11-22 at 11 42 01 AM](https://user-images.githubusercontent.com/22112888/203384487-9ec954bf-477c-4116-b19a-aa5b6499d05f.png)

Reply:
With only auto margin, this is how it's getting rendered for me:

<img width="826" alt="image" src="https://user-images.githubusercontent.com/4306604/203403354-81585307-792d-4064-9e27-1f6e3fcef461.png">

Have you tried refreshing without cache?

Reply:
Hi, @minump, Could you please look into this when you get a chance?

Reply:
11px seemed to be aligned towards the bottom for me. Maybe 7px? See attached screenshot for 7px. Updated css style margin for 7px.
<img width="1260" alt="Screen Shot 2022-11-29 at 4 51 03 PM" src="https://user-images.githubusercontent.com/22112888/204665927-bdb52991-1fcb-4565-870f-84324e1544b1.png">

Reply:
Okay. Looks close. Thanks.